rd-cpp/thirdparty/clsocket/src/StatTimer.h

```diff
@@ -62,6 +62,8 @@
 
 #include "Host.h"
 
+#include <algorithm>
```

Comment:
_(Minor)_ Why do we need `<algorithm>` here?

Reply:
I believe I left it after one of refactoring passes. Can be removed.